packages/ui-library/src/components/stepper/stories.js

```diff
@@ -1,29 +1,49 @@
 import React, { useState, useCallback } from "react";
-import { useArgs } from "@storybook/preview-api";
 import { Stepper } from ".";
-import { component } from "./docs";
+import { component, customStep } from "./docs";
 import { Button } from "../../index";
+import { CustomStep } from "./custom-step";
+
+
+export default {
+	title: "2) Components/Stepper",
+	component: Stepper,
+	argTypes: {
+		className: { control: "text" },
+		currentStep: { control: "number" },
+		steps: { control: "object" },
+		children: { control: "text" },
+	},
+	parameters: {
+		docs: {
+			description: { component },
+		},
+	},
+	args: {
```

Comment:
These influence _all_ the stories in this file.

You could add margin-top to the button instead?

Reply:
Good point